docs/dev/meid_quota.rst

```diff
@@ -0,0 +1,25 @@
+.. _meid-quota:
+
+Update the modelable_entity_id quota in the db
+===============================================
+
+By default the modelable_entity_id can have at most 10 models defined.
+
+This cap is stored in the db table epi.modelable_entity_quota as quota.
+
+The quota can be updated by calling the stored procedure, epi_model_quota_add.
+
+As an example, this SQL CALL statement will change the quota value for meid=1744 to 20.
```

Comment:
Put two colons at end so the next line gets to be code. And remove spaces between lines in these paragraphs, except around code.

Reply:
changed.  These look good in the html.